blocks/header/header.css

```diff
@@ -1,809 +1,265 @@
-    margin: 0 calc((100% - 714px) / 2);
-    max-width: 714px;
-  }
-}
-
-@media (min-width: 992px) {
-  header nav #div-widget-id-search-input {
-    width: 290px;
-  }
-
-  header nav #div-widget-id-autosuggest-results {
-    left: unset;
-    width: 290px;
-    top: 101px;
-    margin: 0;
-  }
 }
 
-header nav #div-widget-id-autosuggest-results .searchstudio-active-row {
-  background: #f2f2f2;
+.header__main-nav-item {
+  border-bottom: solid 1px var(--c-grey-300);
+  padding: 5px 0;
 }
 
-/* hamburger menu: tools */
-header nav .tools {
-  display: flex;
-  align-items: center;
+.header__main-nav-link {
+  padding: 12px 0;
+  border-bottom: solid 2px transparent;
   justify-content: space-between;
-  position: fixed;
-  width: 97vw;
-  height: 73px;
-  right: -97vw;
-  top: 0;
-  z-index: 1052;
-  background-color: white;
-  transition: right 0.2s cubic-bezier(0.4, 0, 0.2, 1) 0.1s;
-  border-bottom: 1px solid var(--c-grey-1);
 }
 
-header nav .hamburger-toggle[aria-expanded='true'] ~ .tools {
-  right: 0;
-}
-
-/* list of tools */
-header nav .tools ul {
-  display: flex;
-  list-style-type: none;
-  margin: 0;
-  padding: 0 0 0 15px;
-  flex: 1;
+.header__main-nav-link--collapsed .icon {
+  transform: rotate(-90deg);
 }
 
-/* tools entries */
-header nav .tools ul li {
+.header__link:any-link {
   display: flex;
-  flex-flow: column;
   align-items: center;
-  justify-content: space-between;
-  margin-left: 4%;
-}
-
-header nav .tools ul li:first-child {
-  margin-left: 0;
-}
-
-header nav .tools ul li a {
-  color: var(--c-grey-4);
-  text-decoration: none;
-  font-size: 8pt;
+  gap: 4px;
+  color: var(--c-grey-700);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-family: var(--ff-volvo-novum);
+  font-size: 12px;
+  line-height: 125%;
+  letter-spacing: 0.4px;
+  border-bottom: solid 2px transparent;
+  padding: 12px 0;
 }
 
-header nav .tools ul li img {
-  width: 16px;
-  height: 16px;
+.header--hamburger-open .header__link:hover,
+.header--hamburger-open .header__link:focus,
+.header--hamburger-open .header__link:active {
+  border-bottom: solid 2px transparent;
 }
 
-/* close icon */
-header nav .tools .hamburger-close {
-  padding: 0 15px;
+.header__link:any-link:active {
+  color: var(--c-main-black);
 }
 
-header nav .tools .hamburger-close img {
-  width: 14px;
-  height: 14px;
+.header__link .icon,
+.header__link svg {
+  height: 24px;
+  width: auto;
 }
 
-/* hamburger menu */
-header nav .sections {
+.header__actions {
   display: flex;
-  align-items: start;
-  justify-content: start;
-  position: fixed;
-  width: 97vw;
-  height: calc(100% - 73px);
-  right: -97vw;
-  top: 73px;
-  z-index: 1052;
-  background-color: white;
-  transition: right 0.2s cubic-bezier(0.4, 0, 0.2, 1) 0.1s;
-  overflow: hidden scroll;
+  gap: 24px;
 }
 
-header nav .hamburger-toggle[aria-expanded='true'] ~ .sections {
-  right: 0;
-}
-
-/* section entries */
-header nav .sections ul {
+.header__actions-list {
```

Comment:
Lots of styles are shared btw `.header__actions-list` and `.header__main-links`, can we put them once?

Reply:
Yup right, fixed